x-pack/plugins/observability_solution/apm/public/components/app/alerts_overview/index.tsx

```diff
@@ -75,6 +75,14 @@ export function AlertsOverview() {
     [history]
   );
 
+  const alertsTableFeatureIds = [
+    AlertConsumers.APM,
+    AlertConsumers.OBSERVABILITY,
+    AlertConsumers.SLO,
+    AlertConsumers.LOGS,
+    AlertConsumers.INFRASTRUCTURE,
```

Comment:
What about uptime?

Reply:
Why should we include uptime here? It's not part of the [alert indexes we are using as guide](https://github.com/elastic/kibana/blob/3e899e748e03000eef1712dd65e01d8fc6e99930/x-pack/plugins/observability_solution/inventory/server/lib/create_alerts_client.ts/create_alerts_client.ts#L21C3-L27C6) either.

Reply:
Anyway, uptime was deprecated in 8.15.0 so we might not need to worry about it.

Reply:
This issue is focused on ensuring consistency between the Entity Inventory and APM, so the changes we've made should cover that. 

However, if you think we need to review the Entity Inventory alert indexes separately, we can create a new issue for it and update what’s necessary in APM and Infra too. That said, I'm not sure if it's worth spending time on it given the deprecation you mentioned.